.github/workflows/ci.yml

```
@@ -18,24 +18,25 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 60
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
       with:
         submodules: recursive
-    - uses: risc0/risc0/.github/actions/rustup@release-0.18
-    - uses: risc0/risc0/.github/actions/sccache@release-0.18
-    - run: cargo install cargo-risczero && cargo risczero install
+    - uses: risc0/risc0/.github/actions/rustup@release-0.19
+    - uses: risc0/cargo-install@v1
+      with:
+        crate: cargo-binstall
+    - run: cargo binstall -y cargo-risczero
+    - run: cargo risczero install
```

To prevent future breakages, I recommend pinning versions like:
```suggestion
    - run: cargo binstall -y cargo-risczero@0.19.1
    - run: cargo risczero install --version test-release-2
```

See: https://github.com/risc0/risc0/blob/main/.github/workflows/release.yml#L68